wallet/txs/p-chain/convertSubnetToL1Tx.go

```diff
@@ -5,32 +5,35 @@ package txs
 import (
 	"fmt"
 
-	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/constants"
 
 	avagoTxs "github.com/ava-labs/avalanchego/vms/platformvm/txs"
 )
 
 // ConvertSubnetToL1TxParams contains all parameters needed to create a ConvertSubnetToL1Tx
 type ConvertSubnetToL1TxParams struct {
 	// SubnetAuthKeys are the keys used to sign `ConvertSubnetToL1Tx`
-	SubnetAuthKeys []ids.ShortID
+	SubnetAuthKeys []string
```

Comment:
Similar question as in the other PR. Switching to string for better UX? To make it even better, we should probably provide examples or state the format. How you commented the example looks helpful.

Reply:
ya string is easier for users to input